Keb region mapping for trial (#171)
kyma-project/control-plane: docs/kyma-environment-broker/03-01-service-description.md

| Parameter name | Type | Description | Required | Possible values| Default value |  
| ---------------|-------|-------------|----------|---------------|---------------|  
| **name** | string | Specifies the name of the Kyma Runtime. | Yes | Any string| None |  
| **region** | string | Defines the cluster region. | No | `europe`,`us`, `asia` | Calculated from the platform region |  
| **provider** | string | Specifies the cloud provider used during provisioning. | No | `Azure`, `GCP` | `Azure` |
 
 </details>
 </div>

> The region is an optional parameter. If not set, the region is calculated from platform region specified in the path:

Reviewer: ```suggestion
The **region** parameter is optional. If not specified, the region is calculated from platform region specified in this path:
```

```shell
/oauth/{platform-region}/v2/service_instances/{instance_id}
```

> The mapping between platform region and the provider (Azure or GCP) region is defined in a configuration file (**APP_TRIAL_REGION_MAPPING_FILE_PATH** environment variable).

Reviewer: ```suggestion
The mapping between the platform region and the provider region (Azure or GCP) is defined in the configuration file in the **APP_TRIAL_REGION_MAPPING_FILE_PATH** environment variable.
```
